service/matching/matching_engine.go

```diff
@@ -958,55 +958,78 @@ func (e *matchingEngineImpl) DescribeTaskQueue(
 			defaultBuildId := getDefaultBuildId(userData.GetVersioningData().GetAssignmentRules())
 			req.Versions = &taskqueuepb.TaskQueueVersionSelection{BuildIds: []string{defaultBuildId}}
 		}
-		// collect internal info
+		rootPM, _, err := e.getTaskQueuePartitionManager(ctx, rootPartition, true, loadCauseDescribe)
+		if err != nil {
+			return nil, err
+		}
+		rootPMImpl, ok := rootPM.(*taskQueuePartitionManagerImpl)
+		if !ok {
+			return nil, fmt.Errorf("failed type assertion on root partition")
+		}
+
+		lastFanOut := rootPMImpl.timeSinceLastFanOut()
```

Comment:
I think the later code would be more clear if you renamed `lastFanOut` --> `timeSinceLastFanOut`